packages/web-components/fast-components/src/tooltip/tooltip.styles.ts

```diff
@@ -0,0 +1,94 @@
+import { css } from "@microsoft/fast-element";
+import { forcedColorsStylesheetBehavior } from "@microsoft/fast-foundation";
+import {
+    accentFillActiveBehavior,
+    accentFillHoverBehavior,
+    accentFillRestBehavior,
+    neutralFillHoverBehavior,
+    neutralFillInputActiveBehavior,
+    neutralFillInputHoverBehavior,
+    neutralFillInputRestBehavior,
+    neutralFillRestBehavior,
+    neutralFocusBehavior,
+    neutralForegroundRestBehavior,
+    neutralOutlineRestBehavior,
+} from "../styles/index";
+
+export const TooltipStyles = css`
+    :host {
+        contain: layout;
+        overflow: visible;
+        height: 0;
+        width: 0;
+    }
+
+    .tooltip {
+        box-sizing: border-box;
+        border-radius: calc(var(--corner-radius) * 1px);
+        border: calc(var(--outline-width) * 1px) solid ${accentFillActiveBehavior.var};
+        border-color: ${neutralFocusBehavior.var};
+        box-shadow: 0 0 0 1px ${neutralFocusBehavior.var} inset;
+        background: ${neutralFillRestBehavior.var};
+        color: ${neutralForegroundRestBehavior.var};
+        padding: 4px;
+        height: fit-content;
+        width: fit-content;
+        font: inherit;
+        font-size: var(--type-ramp-base-font-size);
+        line-height: var(--type-ramp-base-line-height);
+        white-space: nowrap;
+        z-index: 10000;
+    }
+
+    fast-anchored-region {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        overflow: visible;
+    }
+
+    :host(.top) fast-anchored-region,
+    :host(.bottom) fast-anchored-region {
+        flex-direction: row;
+    }
+
+    :host(.right) fast-anchored-region,
+    :host(.left) fast-anchored-region {
+        flex-direction: column;
+    }
+
+    :host(.top) .tooltip {
+        margin-bottom: 4px;
+    }
+
+    :host(.bottom) .tooltip {
+        margin-top: 4px;
+    }
+
+    :host(.left) .tooltip {
+        margin-right: 4px;
+    }
+
+    :host(.right) .tooltip {
+        margin-left: 4px;
+    }
```

Comment:
Will these need to localize? Perhaps instead of left/right we should use language similar to the CSS logical properties and map to those? Curious on thoughts from @EisenbergEffect, @nicholasrice, others?

```
:host(.inline-start) {
    margin-inline-start: 4px;
}
```

Reply:
That makes good sense to me. If that's not enough we also have the new RTL behaviors that Nick wrote. Those are going to be more heavy weight than a standard CSS property though. Just mentioning it for completeness.

Reply:
For sure - I like the logical aspect of the naming; we did something similar with the start/end slots.

Reply:
The current configuration works in rtl.  The host "left"/"right" classes are applied based on the actual position of the region relative to the anchor.  If the region is left we want a margin on the right and vice versa.

Reply:
Anchored region supports "start/end" as a choice for horizontal position when authors want position to change based on direction.  I shall add that as a position choice as well as "left/right".